source/december-2019-template.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Countdown to The New Year - XYour Addon's Name In Title CaseX
```

Comment:
lgtm! 

nit - feel free to ignore - been thinking about this re: Ember Times ... I'm beginning to wonder if it makes more sense to list addons in lowercase as they are on github e.g. `ember-svg-jar` or `prettier`

Reply:
Yeah I agree as well! I think in a previous PR we discussed that [in the scope of the December series, we'd like to title case in the actual title of the post, but use the lowercase version everywhere else](https://github.com/ember-learn/ember-blog/pull/377#discussion_r351007857) - does this make sense and what are your thoughts?

Reply:
@jessica-jordan 👍 sgtm, thanks for the context